picard/formats/id3.py

```diff
@@ -330,6 +333,20 @@ def _load(self, filename):
                 if frame.desc:
                     name += ':%s' % frame.desc
                 metadata.add(name, frame.text)
+            elif frameid == 'SYLT':
+                name = 'syncedlyrics'
+                if frame.desc:
+                    name += ':%s' % frame.desc
+
+                lrc_lyrics = []
+                for lyrics, milliseconds in frame.text:
+                    minutes = milliseconds // (60 * 1000)
+                    seconds = (milliseconds % (60 * 1000)) // 1000
+                    hundredths = (milliseconds % 1000) // 10
+                    lrc_lyrics.append(f"[{minutes:02d}:{seconds:02d}.{hundredths:02d}]{lyrics.strip()}")
+                lrc_lyrics = "\n".join(lrc_lyrics)
```

Comment:
We should move this code to a separate method in order to ease testing.

Something like:

```python
def parse_sylt_text(text):
    for lyrics, milliseconds in text:
        minutes = milliseconds // (60 * 1000)
        seconds = (milliseconds % (60 * 1000)) // 1000
        hundredths = (milliseconds % 1000) // 10
        yield f"[{minutes:02d}:{seconds:02d}.{hundredths:02d}]{lyrics.strip()}"
```

```python
lrc_lyrics = "\n".join(parse_sylt_text(frame.text))
```

This way, unit tests can be written for `parse_sylt_text()`